src/main/java/coverage2/FileOutputDemo.java

```diff
@@ -0,0 +1,31 @@
+package coverage2;
+
+import java.io.*;
+
+class FileOutputDemo 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1118)